composer.json

```diff
@@ -5,6 +5,7 @@
   "type": "library",
   "require": {
     "php": ">=8.2",
+    "laravel/framework": ">=9.0",
```

Comment:
Not sure whether this change is related to the rest of the code in this PR, or if there is another reason to include this dependency now?

Reply:
Some code in this package (service providers) depend on the Laravel framework being installed. It worked previously, because this repo has `laravel/framework` listed as a nested dependency through the `myparcelcom/json-api` package. But if this package uses its contents, it should have it listed as a primary dependency.